### PR TITLE
# Updated to new Version with Python 3

### PR DESCRIPTION
**Pull request by chainstair**

### CODE REVIEW

**Comment by wiejakp** [comment]
@chainstair, thank you for the update. Your changes look good and I'll include them in next release.

**Review by wiejakp — CHANGES_REQUESTED** [review]
2 things, you left out debugging message and  bug in your repo is true.

though can't merge repo as is, i will be using your code for v0.4.

my apologies not sure how to make a fast update to a pr from another repo :)

File: python-sitemap-generator.py [hunk 1]
```diff
@@ -39,7 +43,7 @@
 MaxSubThreads = 10
 
 # DWFINE YOUR URL
-InitialURL = 'URL'
+InitialURL = 'http://test.at/'
```
Comment (wiejakp) [inline python-sitemap-generator.py:46]: change isn't needed

File: python-sitemap-generator.py [hunk 2]
```diff
@@ -1,22 +1,26 @@
 #!/usr/bin/python
 
 # Python Sitemap Generator
-# Version: 0.2
+# Version: 0.4
+
+# Updated to new Version with Python 3
```
Comment (wiejakp) [inline python-sitemap-generator.py:6]: change should go in changelog instead.

**Review by wiejakp — CHANGES_REQUESTED** [review]

**Comment by chainstair** [comment]
You r welcome to use my code :)

Best regards
chainstair

**Pull request merged**